### PR TITLE
Add speaker: Shubham Padia

### LINKED ISSUE #305: add speaker Shubham Padia

### PR DESCRIPTION
**Pull request by shubham-padia**
fixes #305

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2471,6 +2471,25 @@ <h1>Speakers</h1>
                                 <span>KDE</span>
                             </div>
                         </div>
+
+                        <div class="col-md-3 col-sm-6 speaker-column">
+                            <div class="speaker">
+                                <div class="image-holder">
+                                    <img class="background-image" alt="Shubham Padia" src="img/shubhamp.jpg">
```
Comment (simsausaurabh) [inline index.html]: @shubham-padia please use ShubhamPadia.jpg. It's the followed convention. :smile:
Please also provide preview link. Thanks

Comment (shubham-padia) [reply index.html]: done

**Comment by shubham-padia** [comment]
Preview: http://shubhamp.in/2018.fossasia.org/

**Comment by mariobehling** [comment]
@shubham-padia Please help to review the incoming PRs as well. Cheers.

**Comment by shubham-padia** [comment]
@mariobehling sure

**Pull request merged**